### PR TITLE
Feature/add config

### PR DESCRIPTION
**Pull request by MKMonkeyCat**
# Description

add config
1. Build faster
2. add prettieri config
3. fix build Infinity for chrome

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: config/constant.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * @name Config
+ * @description 配置
+ */
+
+/**移除 build console */
```
Comment (SiongSng) [inline config/constant.ts]: Please use English comments

File: vite.config.ts [hunk 2]
```diff
@@ -11,4 +12,17 @@ export default defineConfig({
       '@': fileURLToPath(new URL('./src', import.meta.url)),
     },
   },
+  build: {
+    terserOptions: {
+      compress: {
+        // Prevent Infinity from being compressed to 1/0, may cause performance issues on Chrome
+        keep_infinity: true,
+        drop_console: VITE_DROP_CONSOLE,
```
Comment (SiongSng) [inline vite.config.ts]: Just set `VITE_DROP_CONSOLE` to true

**Review by SiongSng — CHANGES_REQUESTED** [review]

File: vite.config.ts (hunk 2 above)
Comment (qmkcat) [reply vite.config.ts]: https://github.com/Lipoic/Lipoic-Frontend/pull/35/commits/d40e8415b5dc066af086737a87b8fc1ca6270039

File: config/constant.ts (hunk 1 above)
Comment (qmkcat) [reply config/constant.ts]: https://github.com/Lipoic/Lipoic-Frontend/pull/35/commits/d40e8415b5dc066af086737a87b8fc1ca6270039

**Review by SiongSng — APPROVED** [review]
LGTM!

**Pull request merged**